gen-ai/orchestrator-core/src/main/kotlin/ai/tock/genai/orchestratorcore/mappers/EMSettingMapper.kt

```diff
@@ -60,20 +60,27 @@ object EMSettingMapper {
      * @param botId the bot ID (also known as application name)
      * @param feature the feature name
      * @param dto the [EMSettingDTO]
+     * @param rawByForce force the creation of a raw secret key
      * @return [EMSetting]
      */
-    fun toEntity(namespace: String, botId: String, feature: String, dto: EMSettingDTO): EMSetting =
+    fun toEntity(
+        namespace: String,
+        botId: String,
+        feature: String,
+        dto: EMSettingDTO,
+        rawByForce: Boolean = false
+    ): EMSetting =
         with(dto){
             when(this){
                 is OpenAIEMSetting ->
                     OpenAIEMSetting(
-                        apiKey = SecurityUtils.createSecretKey(namespace, botId, feature, apiKey),
+                        apiKey = SecurityUtils.createSecretKey(namespace, botId, feature, apiKey, rawByForce),
```

Comment:
I don't see exactly where this rawByForce is used, I understand why you added it to bypass the genAISecretManagerProvider.

Reply:
heare : https://github.com/theopenconversationkit/tock/pull/1834/files#:~:text=%3D%20request.llmSetting%2C-,rawByForce%20%3D%20true,-)%2C

Reply:
Thanks